docker/root/usr/bin/setup

```diff
@@ -41,12 +44,7 @@ if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
     REQUIRE_SIGNIN_VIEW=${REQUIRE_SIGNIN_VIEW:-"false"} \
     SECRET_KEY=${SECRET_KEY:-""} \
     envsubst < /etc/templates/app.ini > ${GITEA_CUSTOM}/conf/app.ini
-
-    chown ${USER}:git ${GITEA_CUSTOM}/conf/app.ini
 fi
 
-# only chown if current owner is not already the gitea ${USER}. No recursive check to save time
-if ! [[ $(ls -ld /data/gitea | awk '{print $3}') = ${USER} ]]; then chown -R ${USER}:git /data/gitea; fi
-if ! [[ $(ls -ld /app/gitea  | awk '{print $3}') = ${USER} ]]; then chown -R ${USER}:git /app/gitea;  fi
-if ! [[ $(ls -ld /data/git   | awk '{print $3}') = ${USER} ]]; then chown -R ${USER}:git /data/git;   fi
-chmod 0755 /data/gitea /app/gitea /data/git
+chmod 0755 /data/gitea /data/git
+#chmod 0755 /app/gitea
```

Comment:
`/app/gitea` is owned by root.
```suggestion
```

Reply:
Yes and i don't think it is a problem. I will recheck but that a good point that the process inside the containre couldn't change content inside /app/gitea